pkg/crc/machine/machine.go

```diff
@@ -253,6 +253,15 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	}
 	logging.Info("CodeReady Containers VM is running")
 
+	// Stop Chronyd service
+	if startConfig.StopChronyd {
+		logging.Info("Stopping chronyd service in CodeReady Containers VM")
+		if _, err := sshRunner.Run("sudo systemctl stop chronyd"); err != nil {
```

Comment:
This feels more low-level than `sudo timedatectl set-ntp off`, any reason to directly interact with `chronyd`?

Reply:
@cfergeau I didn't check `timedatectl set-ntp off ` but looking at the man page looks like it also start/stop underline service in timedatectl side it is `systemd-timesyncd.service` or `chronyd` so I am not sure if it have any improvement over how we are interacting.

```
set-ntp [BOOL]
           Takes a boolean argument. Controls whether network time synchronization is active and enabled (if available). If the argument is true, this enables and starts the first existing network synchronization service. If the
           argument is false, then this disables and stops the known network synchronization services. The way that the list of services is built is described below.

$ timedatectl set-ntp true
           ==== AUTHENTICATING FOR org.freedesktop.timedate1.set-ntp ===
           Authentication is required to control whether network time synchronization shall be enabled.
           Authenticating as: user
           Password: ********
           ==== AUTHENTICATION COMPLETE ===

           $ systemctl status systemd-timesyncd.service
           ● systemd-timesyncd.service - Network Time Synchronization
              Loaded: loaded (/usr/lib/systemd/system/systemd-timesyncd.service; enabled)
              Active: active (running) since Mo 2015-03-30 14:20:38 CEST; 5s ago
                Docs: man:systemd-timesyncd.service(8)
            Main PID: 595 (systemd-timesyn)
              Status: "Using Time Server 216.239.38.15:123 (time4.google.com)."
              CGroup: /system.slice/systemd-timesyncd.service
                      └─595 /usr/lib/systemd/systemd-timesyncd

```

Reply:
> @cfergeau I didn't check `timedatectl set-ntp off ` but looking at the man page looks like it also start/stop underline service in timedatectl side it is `systemd-timesyncd.service` or `chronyd` so I am not sure if it have any improvement over how we are interacting.

This shields us from the NTP service changing in the future, if there's a switch in the future from `chronyd` to `systemd-timesyncd`, `timedatectl` command line will be unchanged, while `systemctl stop chronyd` would stop working and we'd need to change our code.